### DISCUSSION TITLE
Athens School of Design

### DISCUSSION THREAD

**Discussion by jsmorabito** [athensresearch/athens#1233] [General]
What would you like to see in a Clojurefam analog on the design side of the Athens project?

**Comment by Ejipuh** [comment]
Seeing that a lot of design seems to center around UX thinking, this would serve as a good starting point to the School of Design. Anyone can learn to analyze their common tools and apps, and making that skill accessible will help improve Athens, if not software in general through raised awareness. 

Some things it could delve into:
- What good design is: how to evaluate everyday things you use, apps, and games--anything interactive. In other words, becoming skilled at asking the purpose of things.
- Guidelines for choosing tools like a craftsman, how to critique apps, how to submit a stellar feature request (understanding the impact that different design choices would make on a system as a whole)

Potential Assignments
- Analyze a non-digital tool you use every day. What purpose does it intend to serve? Does it do this well? If not, what should be improved about it? Is it enjoyable to use? If not, why? Etc.
- Analyze an app you often use. What does it do? What behaviors or ways of thinking is it guiding you toward? Is it actually benefiting you? What is good about it? What should be improved about it? If you had a magic wand and the developer's approval, where would you take it and why? Etc.
- Analyze Athens as a tool. (See above questions.) 
- Identify a novel UX problem with Athens, and come up with a new feature request that addresses it. Work with a programmer, and gauge how feasible implementing your feature idea would be. (This might require some knowledge of how the system works.)

For more general structure and setup, @seekanddefine made a good point that mentorships are incredibly valuable. These assignments could be submitted to a mentor, who would gauge how well you as a student understand the core concepts of design. In the meanwhile, you would be engaging the community and larger PKM space, asking questions about these topics and spreading design-awareness. Perhaps starting a blog could be part of this?

Anyways, these are what I have in mind now. I'd appreciate feedback or further ideas.
